src/server.c

```diff
@@ -4875,6 +4886,10 @@ void addReplyCommandDocs(client *c, struct redisCommand *cmd) {
         addReplyBulkCString(c, "history");
         addReplyCommandHistory(c, cmd);
     }
+    if (cmd->reply_schema) {
+        addReplyBulkCString(c, "reply_schema");
+        addReplyJson(c, cmd->reply_schema);
+    }
```

Comment:
We decided that for now, we'll avoid adding any interface changes.
we'll use the tests to validate the schema, but keep it internal to be exposed in some future date.
so this needs an `#ifdef`